website/static/js/home-page/newAndNoteworthyPlugin.js

```diff
@@ -43,12 +43,17 @@ var NewAndNoteworthy = {
         var newAndNoteworthyUrl = $osf.apiV2Url('nodes/' + window.contextVars.newAndNoteworthy + '/node_links/', {});
         var newAndNoteworthyPromise = m.request({method: 'GET', url: newAndNoteworthyUrl, config: xhrconfig, background: true});
         newAndNoteworthyPromise.then(function(result){
-            var numNew = Math.min(result.data.length, self.SHOW_TOTAL);
+            var numNew = result.data.length;
             for (var l = 0; l < numNew; l++) {
                 var data = result.data[l];
                 if (lodashGet(data, 'embeds.target_node.data', null)) {
-                    self.newAndNoteworthyNodes().push(result.data[l]);
-                    self.fetchContributors(result.data[l]);
+                    if (data.embeds.target_node.data.attributes.public === true) {
```

Comment:
So if you made your project private, it wouldn't show up for me in the Discover Public Projects list, but it would still show up for you.  QA pointed this out - then you might think everyone else can see the project you decided to make private.